common/src/main/java/org/apache/drill/common/PlanStringBuilder.java

```diff
@@ -56,20 +84,30 @@ public PlanStringBuilder field(String key, Object value) {
     return this;
   }
 
+  /**
+   * Displays a numeric field: {@code size=10}.
+   */
   public PlanStringBuilder field(String key, int value) {
     startField(key);
     buf.append(value);
     return this;
   }
 
+  /**
+   * Displays a character in Java-quoted format: {@code delimiter="\n"}.
+   * @param key
```

Comment:
Please add descriptions, sorry for asking that, it's just IDE warning are annoying when descriptions are not added.